solidity/dashboard/package.json

```diff
@@ -16,7 +16,7 @@
     "formik": "^2.1.3",
     "moment": "^2.20.1",
     "web3": "^1.2.4",
-    "@keep-network/keep-core": ">0.13.0-pre <0.13.0-rc"
+    "@keep-network/keep-core": "0.13.0-rc.0"
```

Comment:
This should just be `-rc`, fwiw.

Also I guess maybe it should be `>0.14.0-pre <0.14.0-rc`?

Reply:
Ah, but you're worried this'll fail when we run it because we don't have published 0.14.0-pre artifacts for keep-core.

Reply:
Yes, I wanted to change it to `>0.14.0-pre <0.14.0-rc` once we have them published.